src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,108 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import React from 'react';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (newValue: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
   const [count] = useState(0);
+  const [valueTitle, setValueTitle] = useState('');
+  const [valueDescription, setValueDescription] = useState('');
+  const [valueImgUrl, setValueImgUrl] = useState('');
+  const [valueImdbUrl, setValueImdbUrl] = useState('');
+  const [valueImdbId, setValueImdbId] = useState('');
```

Comment:
use one state instead of 5
```suggestion
  const [movie, setMovie] = useState({
      title: '',
      description: '',
      imgUrl: '',
      imdbUrl: '',
      imdbId: '',
    });
```